contents/handbook/strategy/overview.md

```diff
@@ -140,24 +142,37 @@ While there is value in the items on the right, we value the items on the left m
 
 * _Reject_: We enable our customers to ingest, store and analyze data on their infrastructure, we don't believe tons of integrations, dealing with multiple vendors and sending sensitive data to multiple cloud providers is the right approach. Providing all the apps and data platform in one place, removes the need to setup a warehouse in the first place.
 
-## Direction for 2022
-
- * 2 word summary: **Nail self-serve**
-    * **Customers**
-        * **Focus on high-growth startups.** E.g. Large initial contracts ($20k-$70k/year) and smaller deals in organizations that will eventually become large
-        * **Non Goal:** Start doing outbound sales
-    * **Product**
-        * **Goals:**
-            * **Quality:** Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
-            * **Self Service Subscription:** Customers of every size sign-up, start using and subscribe to PostHog without asking us for help
-        * **Non Goals:**
-            * Build lots of new low quality / partial features
-            * Build everything a single enterprise customer wants just to close a deal
-
-## Target customers for 2022
-
-We build for the highest-performing **product teams** (engineers, PMs, designers) building the **most loved products** at **high-growth startups**. We focus specifically on the  **product-engineers** (full-stack engineers skewed towards the frontend) but it should be usable by everyone in the product team.
+## Direction for 2023
+
+3-word summary: **Nail product engineering**
```

Comment:
I would like to make the revolutionary proposal to not use the "nail" terminology anymore, at least not in this context. If we want to stick with a 3-word summary, I would tend to something like "Supercharge Product Engineers", because I think that is a nicer visualisation, imagining how a Product Engineer will _excel_ in their work because of PostHog.

Reply:
genuine q - on either suggestion... will this differ in say 2024? i am wondering if it should be more specific to this _year's_ approach

Reply:
Good question - I think this 3-word summary represents a reasonably big shift compared to the focus previously (nail self-serve). In terms of future looking, if we execute well on this I could see either broadening the summary, switching the ICP for a few quarters (e.g. nail growth engineering), or choosing a more specific part of product engineering side that we don't think we have nailed yet. However we might also want to roll it over for another few quarters if we don't think we have it nailed yet.

I'm happy with "Supercharge product engineers" as is but happy to hear if you have a better suggestion which encapsulates what we are wanting to do